packages/client-search/src/types/GetDictionarySettingsResponse.ts

```diff
@@ -0,0 +1,11 @@
+import { DictionaryName } from './DictionaryName';
+import { RequireAtLeastOne } from './RequireAtLeastOne';
+
+export type GetDictionarySettingsResponse = {
+  /**
+   * TODO: Description
```

Comment:
```suggestion
   * Disable the builtin Algolia entries for a type of dictionary per language
```

something like that?

Reply:
Oops I forgot about these, make sense!